dandiapi/api/management/commands/migrate_published_version_metadata.py

```diff
@@ -21,10 +21,6 @@ def migrate_published_version_metadata(dandiset: str, published_version: str, to
     )
     metadata = version.metadata
 
-    # If there is no schemaVersion, assume to_version
-    if 'schemaVersion' not in metadata:
-        metadata['schemaVersion'] = to_version
-
```

Comment:
is this being removed for a specific reason? or because we no longer have such a possibility?

Reply:
The latter, since attempting to exclude `schemaVersion` from the metadata would fail at the database level.